### ISSUE TITLE
Symfony 5.2: Container App_KernelDevDebugContainer.xml does not exist

### ISSUE THREAD

**Issue by tristanbes** [phpstan/phpstan-symfony#112]
Using Symfony 5.2, the container is no longer dumped when using `cache:clear` or `cache:warmup`.
A possible workaround is to use `bin/console lint:container` to create the `

This issue is to help user track the problem since it's Symfony related (I guess): https://github.com/symfony/symfony/issues/39240

I had to use that in my CI:
```yaml
-   run: |
        APP_ENV=dev php bin/console cache:clear
        APP_ENV=dev php bin/console cache:warmup
        APP_ENV=dev php bin/console lint:container # Or the App_KernelDevDebugContainer.xml does not exist
        php bin/phpstan analyse --no-progress --error-format=checkstyle | cs2pr
    working-directory: ${{ matrix.working_directory }}
    if: always()
```

**Comment by ondrejmirtes** [comment]
FYI `php bin/phpstan analyse --no-progress --error-format=checkstyle | cs2pr` can be simplified to `php bin/phpstan analyse --no-progress`

**Comment by tristanbes** [comment]
thanks for the tip ! I guess I missed the support for github annotation in your changelog if you recommend dropping `cs2pr` 👍

**Comment by mhujer** [comment]
It is resolved in Symfony 5.2.1 - see https://github.com/symfony/symfony/pull/39360 (linked from the original issue)
